addons/spreadsheet/utils.py

```diff
@@ -1,26 +1,97 @@
 import json
 import base64
+import re
 
 from odoo import _
 
-def empty_spreadsheet_data_base64():
+def empty_spreadsheet_data_base64(lang):
```

Comment:
you can move `empty_spreadsheet_data_base64` and `empty_spreadsheet_data` to `spreadsheet.mixin` methods.
That way you can remove the parameter and all those `_get_lang` before every call